plans/artifactory-pro/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+echo "Starting Artifactory Pro"
+
+export JAVA_HOME=$(hab pkg path core/server-jre)
+export ARTIFACTORY_HOME={{pkg.svc_var_path}}
+
+{{pkg.svc_var_path}}/bin/artifactory.sh 2>&1
```

Comment:
I'd say you should make a run hook if the thing gets longer that what should fit on one line, as a loose, subjective, guideline.

`pkg_svc_run` is nice for plans that don't have any other hooks because you only need to have the plan.sh file, but if you already have a hooks directory, and your run hook is more than one line, put it in a separate hook.